webob/response.py

```diff
@@ -256,11 +260,6 @@ def _status__set(self, value):
             raise TypeError(
                 "You must set status to a string or integer (not %s)"
                 % type(value))
-        if ' ' not in value:
-             try:
-                value += ' ' + status_reasons[int(value)]
-             except KeyError:
-                value += ' ' + status_generic_reasons[int(value) // 100]
```

Comment:
We are losing some validation here I think.

`response.status = 'InvalidStatusCodeString'` will pass here and be set when it should fail, right?